src/devices/bus/nes/bootleg.cpp

```diff
@@ -1186,6 +1255,85 @@ uint8_t nes_09034a_device::read_m(offs_t offset)
 	return m_prg[((m_reg + 4) * 0x2000) + offset];
 }
 
+/*-------------------------------------------------
+
+ BTL-PALTHENA
+
+ Games: Palthena no Kagami (FDS conversion)
+
+ This board has fixed 8K PRG banks at 0x6000, 0x8000,
+ 0xc000, and 0xe000. The PRG bank at 0xa000 is switchable
+ by writing to the register in the same 0xa000-0xbfff
+ range. What makes the board interesting is the overlaid
+ 8K of RAM with only 6K addressable:
+
+   8K WRAM:           Addr:
+     0x0000-0x11ff      0xc000-0xd1ff
+     0x1200-0x12ff      0x8200-0x82ff
+     0x1300-0x17ff           N/A
+     0x1800-0x18ff      0x6000-0x60ff
+     0x1900-0x19ff           N/A
+     0x1a00-0x1aff      0x6200-0x62ff
+     0x1b00-0x1bff           N/A
+     0x1c00-0x1dff      0x6400-0x65ff
+     0x1e00-0x1eff           N/A
+     0x1f00-0x1fff      0xdf00-0xdfff
+
+ NES 2.0: mapper 539
+
+ In MAME: Supported.
+
+-------------------------------------------------*/
+
+u8 nes_palthena_device::read_m(offs_t offset)
+{
+//  LOG_MMC(("palthena read_m, offset: %04x\n", offset));
+	switch (offset & 0x1f00)
+	{
+		case 0x0000: case 0x0200: case 0x0400: case 0x0500:
+			return m_prgram[offset | 0x1800];
+		default:
+			return m_prg[0x0d * 0x2000 + offset];    // fixed PRG bank
+	}
+}
+
+void nes_palthena_device::write_m(offs_t offset, u8 data)
+{
+	LOG_MMC(("palthena write_m, offset: %04x, data: %02x\n", offset, data));
+	switch (offset & 0x1f00)
+	{
+		case 0x0000: case 0x0200: case 0x0400: case 0x0500:
+			m_prgram[offset | 0x1800] = data;
+			break;
```

Comment:
Could you please put the `case` labels on separate lines?  It would make it much easier to see what it does at a glance.

Reply:
Done, and fixed a merge issue as well.